### PR TITLE
Adding live gitcoin exemplars to bounty creation flow. 

### LINKED ISSUE #2271: as a user, i want linked live gitcoin exemplars in the bounty creation flow
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
as a user, i want linked live gitcoin exemplars in the bounty creation flow

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:  so people know what a good bounty example looks like

### Current Behavior
[comment]: # (Describe what actually happened.)
no examples listed

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
these examplars are linked in the bounty creation flow and on the helpdesk

### PR DESCRIPTION
**Pull request by Anish-Agnihotri**
##### Description

This pull request addresses the addition of Gitcoin Exemplars to the bounty creation flow and the helpdesk (in the form of an external article). 

##### Checklist

- [] linter status: 100% pass - Currently fixing.
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

User Interface

##### Testing

Simple UI changes, so no specific testing is needed. I'm unable to tell but `yapf` seems to have auto linted the `views.py` file. Let me know if that is an issue and I will revert and update.

##### Refers/Fixes
Addresses and fixes #2271.

### CODE REVIEW

**Comment by Anish-Agnihotri** [comment]
Inbuilt version of `yapf` auto-linted the document file every time I staged the commit. Fixed the issue by reverting `view.py` to current origin master and making changes directly from GitHub web in order to bypass forced linting upon commit execution from command line. 

Should fix any merge conflicts and pass all checks.

File: app/retail/views.py [hunk 1]
```diff
@@ -735,9 +735,13 @@ def help(request):
         'url': 'https://medium.com/gitcoin/tutorial-how-to-price-work-on-gitcoin-49bafcdd201e',
         'title': _('How to Price Work on Gitcoin'),
     }, {
-        'img': static('v2/images/help/tools.png'),
-        'url': 'https://medium.com/gitcoin/tutorial-post-a-bounty-in-90-seconds-a7d1a8353f75',
-        'title': _('Post a Bounty in 90 Seconds'),
+         'img': 'https://raw.githubusercontent.com/Anish-Agnihotri/Gitcoin-Exemplars/master/helpImage.png',
+         'url': 'https://github.com/Anish-Agnihotri/gitcoin-exemplars',
```
Comment (owocki) [inline app/retail/views.py]: we should probably move the source of truth for these from https://github.com/Anish-Agnihotri/gitcoin-exemplars to the `gitcoinco` org

same comment applies for all the other links too

Comment (Anish-Agnihotri) [reply app/retail/views.py]: Yes, that would make sense. 

I can revise this PR by including the file structure and .md files to the `docs` folder, and referencing it from there. Or, just let me know a directory that works for you.

Comment (owocki) [reply app/retail/views.py]: sure the repo is https://github.com/gitcoinco/Gitcoin-Exemplars -- i just sent you an invite to write to it!

**Comment by Anish-Agnihotri** [comment]
Made a mistake and accidentally commuted all historic PR changes. Fixing in a new PR.

**Pull request closed without merging**